UI/ChatUI/ChatUI/ChatUI/MainPageViewModel.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Windows.Input;
+using ChatUI.Entities;
+
+namespace ChatUI;
+
+public class MainPageViewModel : INotifyPropertyChanged
```

Comment:
I think we want an example that uses MVUX. If so, please refer to the sample I did few weeks ago.

Reply:
I have moved the whole project to using MVUX